Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -4,6 +4,8 @@ import SwiftUI
 struct DisplayHymnBottomBar: View {
 
     @Binding var dialogBuilder: (() -> AnyView)?
+    @Binding var toggleSoundCloud: Bool
+    @Binding var initiatedSoundCloud: Bool
```

Comment:
What is the benefit of passing all this stuff in and displaying it over the hymn lyrics versus just making it a sheet that comes up? I think that would simplify the code a lot.

just do 

```
        case .soundCloud(let url):
            self.sheet = .soundCloud(url)
```

Reply:
The problem is then we have no way to play soundCloud in the background. As soon as you pull the sheet back down your sound cloud session is over. The benefit of this way, even though admittedly it is not as pretty in the code, is that it makes it easy for someone to seamlessly shift between enjoying listening to the song alongside either singing the lyrics or playing along with guitar.